tools/webpack/development.js

```diff
@@ -46,4 +46,33 @@ module.exports = [
 			} ),
 		],
 	},
+	{
+		...sharedConfig,
+		name: 'react-jsx-runtime',
```

Comment:
I'm going to need someone to check that the built script for this is correct. I'm trying to extract it to its own script and using "React" as an external global variable. I think it's ok now but a confirmation would be great.